spring-cloud-kubernetes-integration-tests/spring-cloud-kubernetes-client-config-it/src/test/java/org/springframework/cloud/kubernetes/client/config/it/ConfigMapAndSecretIT.java

```diff
@@ -141,7 +141,8 @@ public void handleError(ClientHttpResponse clientHttpResponse) throws IOExceptio
 		secret.setData(secretData);
 		api.replaceNamespacedSecret(APP_NAME, NAMESPACE, secret, null, null, null);
 		await().timeout(Duration.ofSeconds(60))
-				.until(() -> rest.getForObject(MYSECRET_URL, String.class).equals("p455w1rd"));
+			.pollInterval(Duration.ofSeconds(2))
```

Comment:
I've added `pollInterval(Duration.ofSeconds(2))` in 3 places, without them the logs are just exploding. As an addendum, I really liked the tests here!